packages/babel/package.json

```diff
@@ -13,10 +13,14 @@
   "author": "Rich Harris",
   "homepage": "https://github.com/rollup/plugins/tree/master/packages/babel#readme",
   "bugs": "https://github.com/rollup/plugins/issues",
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "exports": {
+    "require": "./dist/cjs/index.js",
+    "import": "./dist/es/index.js"
+  },
   "engines": {
-    "node": ">= 10.0.0"
+    "node": ">=14.0.0"
```

Comment:
will this PR result in a major bump for `@rollup/plugin-babel`?

Reply:
Yes

Reply:
Also, that is safer considering I am also messing with package.exports